postman/collections/mm2_dev.postman_collection.json

```diff
@@ -4824,6 +4824,45 @@
 						}
 					]
 				},
+				{
+				    "name": "get_current_mtp",
+					"event": [
+						{
+                            "listen": "prerequest",
+                            "script": {
+								"exec": [
+									"// Strip JSON Comments\r",
+									"const rawData = pm.request.body.toString();\r",
+									"const strippedData = rawData.replace(/\\\\\"|\"(?:\\\\\"|[^\"])*\"|(\\/\\/.*|\\/\\*[\\s\\S]*?\\*\\/)/g, (m, g) => g ? \"\" : m)\r",
+									"\r",
+											"pm.request.body.update(strippedData);"
+										],
+										"type": "text/javascript"
+									}
+								}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"userpass\": \"{{userpass}}\",\r\n    \"mmrpc\": \"2.0\",\r\n    \"method\": \"get_current_mtp\",\r\n    \"params\": {\r\n        \"coin\": \"KMD\"\r\n    }\r\n}\r\n"
```

Comment:
We should use test coins whenever possible for this postman collection, please use `RICK` similar to other methods.
You should also add `// "id": null // Accepted values: Integers` optional parameter similar to other v2 methods.

Reply:
done